tests/installation/first_boot.pm

```diff
@@ -64,6 +64,15 @@ sub run {
         handle_login;
     }
 
+    if (is_sle('>=15-SP2') && get_var("STAGING")) {
```

Comment:
do we also have this problem for openSUSE?

Reply:
This problem only appears in restrictive security policy, so it does not appear in openSUSE.

Reply:
this shouldn't be restricted to 15-SP2 staging. The issue is visible with SLES or SLES + WE, but not on SLED nor Leap or TW.